src/utils/web3.ts

```diff
@@ -62,6 +62,9 @@ export function getNetworkDisplayName(
     case 137:
       displayName = 'Polygon'
       break
+    case 246:
+      displayName = 'Energy Web Chain'
```

Comment:
this is the same as on the network metadata so we don't need to overwrite it here. This method is only for those cases where what we get from metadata is weird

Reply:
yeah, I think "Energy Web Chain" sounds better than "energyweb"